### PR TITLE
Feature/boxplots

### PR DESCRIPTION
**Pull request by shukon**

### CODE REVIEW

**Comment by Zethson** [comment]
Picture would be nice. Please clean up your javascript file and remove all console log statements and outcommented code. 
Will then merge in seconds.

File: dark_proteome_visualization/templates/analyze.html [hunk 1]
```diff
@@ -39,8 +39,28 @@
...
+Disorder plays an important role in protein structure, since intrinsically disordered regions are known to account for much of the dark proteome (especially in eukaryotes). Since the obtained visualization had issues and the results are not significant, we briefly sum up the result of the paper regarding disorder of dark proteins:
+Most dark proteins had low disorder (median, 10% disorder) and did not differ too much from non-dark proteins (median, 6% disorder); since both of these medians were less than half of the median darkness score (28%), this result suggests that most of the dark proteome in eukaryotes was not disordered.
+In bacteria, archaea, and viruses, non-dark proteins, surprisingly, had higher median disorder than dark proteins. However, the median darkness was always higher still, implying that in these organisms as well, much of the dark proteome was not disordered.
+</p>
+			<div id="boxplot">
+		<script type="text/javascript"> 
+                var dark_proteomes = {{ dark_proteomes | safe }};
```
Comment (Zethson) [inline dark_proteome_visualization/templates/analyze.html]: Would be nice if we could also reuse the vars like in explore.html

**Comment by shukon** [comment]
> Picture would be nice. Please clean up your javascript file and remove all console log statements and outcommented code.
> Will then merge in seconds.

You are incredibly fast. I love it :D

**Comment by Zethson** [comment]
I'm a :robot:

**Comment by Zethson** [comment]
Oh and you're deleting the mail_stub.conf file.
Likely because you renamed it and it's now picked up by the gitignore. Please rename it again to the mail_stub.conf name. I should have suggested to copy and rename it.

**Comment by Zethson** [comment]
If @modaneliuc is okay with that?
It's her work that will be graded. Currently I don't think that it's clean.

I can merge it if @modaneliuc is fine with it and then she or I can clean it up somewhen later (but not part of our final submission).

**Comment by shukon** [comment]
@Zethson she's ok (says so on slack), merge!

**Comment by shukon** [comment]
Thanks!

**Comment by Zethson** [comment]
Your will is my command.

**Pull request merged**